apps/web/src/components/FeedItem/FeedItem.scss

```diff
@@ -0,0 +1,24 @@
+.feedsbar-feeditem-container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px;
+  gap: 10px;
+}
+
+.feedsbar-feeditem-label {
+  font-family: Verdana;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 19px;
+  letter-spacing: 0em;
+  text-align: left;
+  color: #000000;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+// Remove default underline style for anchor tag link (note: tried applying this style in .feedsbar-feeditem-label and elsewhere but it would not override the style below, even with !important)
+a:-webkit-any-link {
```

Comment:
Have you tested this with other browsers? Targeting the `-webkit` selector may only override the text decoration for Safari?
 
So you'd also want to have:
```scss
a{
  text-decoration: none;
}
```
Also, I think that you're actually okay to target the `a` element vs. being more specific and removing the  underline in a class. We are using `Sass` so all of the CSS is modularized / compartmentalized. So this styling won't target any components that don't directly import the `FeedItem.scss` stylesheet or extend their stylesheet with it. Does that make sense?

What I'm saying is its okay to target top-level HTML elements with Sass / other post-CSS processors in ways that might be more problematic with regular / truly cascading CSS.

(If this isn't clear let's circle back to it in Slack because this is an important thing to understand.)

Reply:
Good call! And that is good to know about the modular nature that Sass brings to the project. It does make sense to me and is super useful to be able to target top level elements without breaking styles in other components.